cassandra/cqlengine/columns.py

```diff
@@ -16,6 +16,8 @@
 from datetime import date, datetime
 import logging
 import six
+import struct
+import iso8601
```

Comment:
ah. Now having in mind that C* has to bundle  all deps, I agree that adding a new dependency just for this might be unnecessary. I'm fine to let this part handled on the application level.